pkpdapp/pkpdapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,80 @@
+                ('name', models.CharField(help_text='name of the biomarker type', max_length=100)),
+                ('unit', models.CharField(choices=[('mg', 'type1')], help_text='units for the value stored in :model:`pkpdapp.Biomarker`', max_length=2)),
+                ('description', models.TextField(help_text='short description of the biomarker type')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Dataset',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='name of the dataset', max_length=100)),
+                ('description', models.TextField(help_text='short description of the dataset')),
+                ('administration_type', models.CharField(choices=[('T1', 'type1'), ('T2', 'type2')], help_text='method of drug administration', max_length=2)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='PkpdModel',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='name of the model', max_length=100)),
+                ('description', models.TextField(help_text='short description of the model')),
+                ('model_type', models.CharField(choices=[('PK', 'Pharmokinetic'), ('PD', 'Pharmodynamic')], help_text='type of model, e.g. PK or PD', max_length=2)),
+                ('sbml', models.TextField(help_text='the model represented using SBML (see http://sbml.org)')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Project',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='name of the project', max_length=100)),
+                ('description', models.TextField(help_text='short description of the project')),
+                ('datasets', models.ManyToManyField(help_text='datasets referenced by this project', to='pkpdapp.Dataset')),
+                ('pkpd_models', models.ManyToManyField(help_text='models referenced by this project', to='pkpdapp.PkpdModel')),
+                ('users', models.ManyToManyField(help_text='users with access to this project', to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Profile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('selected_project', models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to='pkpdapp.Project')),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.CreateModel(
```

Comment:
Do we want to move this up to just follow the Biomarker type model? (Just thinking alphabetically.)

Reply:
not really. I didn't write this, its an automatically generated migration from Django. It will just get overwritten when we next generate it